spec/facades/foods_facade_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Facade' do
+
+  it "returns fods containing a keyword" do
+    foods = FoodsFacade.get_foods('sweet potatoes')
+
+    expect(foods).to be_an(Array)
+    expect(foods).to be_all(Food)
```

Comment:
you can also add an assertion here about how many Food objects are in the array (always 10)